src/networkaccessmanager.cpp

```diff
@@ -365,8 +366,19 @@ QNetworkReply* NetworkAccessManager::createRequest(Operation op, const QNetworkR
     if (!m_localUrlAccessEnabled &&
             (req.url().isLocalFile() || scheme == QLatin1String("qrc"))) {
         reply = new NoFileAccessReply(this, req, op);
+        m_ids[reply] = m_idCounter;
```

Comment:
We're updating this in both branches; looks better to just do it once before the `if`.